src/winetricks

```diff
@@ -4085,6 +4143,8 @@ winetricks_is_installed()
              IFS="$_W_IFS"
             fi
             ;;
+        nix_redox) w_die "Platform '$W_PLATFORM' is not implemented in winetricks_is_installed()" ;;
```

Comment:
Not needed.

Reply:
Same reasoning as https://github.com/Winetricks/winetricks/pull/1338/files/7c0eed4e8a25108c4f9e345657356343a07cbf78#r332380161